demos/tests/cases.py

```diff
@@ -1016,19 +1016,19 @@ def single_option_cases(key, *args):
                           '-m': ModelArg('midasnet')})
     )),
 
-    PythonDemo(name='multi_camera_multi_target_tracking_demo', device_keys=['-d'],
-               model_keys=['-m', '--m_reid'], test_cases=combine_cases(
-        TestCase(options={'--no_show': None,
-            **MONITORS,
-            '-i': [DataPatternArg('multi-camera-multi-target-tracking'),
-                DataPatternArg('multi-camera-multi-target-tracking/repeated')],
-            '-m': ModelArg('person-detection-retail-0013')}),
-        single_option_cases('--m_reid',
-            ModelArg('person-reidentification-retail-0277'),
-            ModelArg('person-reidentification-retail-0286'),
-            ModelArg('person-reidentification-retail-0287'),
-            ModelArg('person-reidentification-retail-0288')),
-    )),
+    #PythonDemo(name='multi_camera_multi_target_tracking_demo', device_keys=['-d'],
```

Comment:
could you please also stay comment about why we exclude test?

Reply:
Added